src/mgmt/cli_mgmt.c

```diff
@@ -371,16 +371,28 @@ get_attach_info(const char *name, int *npsrs, struct dc_mgmt_psr **psrs,
 
 	if (sy_info) {
 		size = sizeof(sy_info->provider);
+		if (strlen(resp->provider) >= size) {
```

Comment:
I think Coverity yells at us if we use strlen instead of strnlen. I had to fix a bunch of these in the semi-recent past.

Reply:
Thanks for the heads up.  I've made the change.